### PR TITLE
Ax 644 start trading button not highlighting fully upon hover

### PR DESCRIPTION
**Pull request by Friendlygeckos**
# Description
Start trading button will now highlight on hover

Fixes Jira Ticket # https://athletex.atlassian.net/browse/AX-644

## Type of change
Please delete options that are not relevant.
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Is this a UI Change? If so please include screenshot of before and after states below
## Before
image

## After
image

# How Has This Been Tested?
Ran the app on chrome using web-server

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have assigned 2 reviewers to check my work
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by pearlson16 — APPROVED** [review]

**Review by durduman — APPROVED** [review]

File: lib/pages/LandingPage.dart [hunk 1]
```diff
@@ -151,39 +150,6 @@ class _LandingPageState extends State<LandingPage> {
...
-              "Reload the App",
-              style: textStyle(Colors.red[400]!, tradingTextSize, true, false),
-            ),
-          ),
-        ),
-      ],
-    );
-  }
-
   Widget startTradingButton(double tradingTextSize) {
```
Comment (narcodico) [inline lib/pages/LandingPage.dart:153]: Please stop using functions to build widgets, unless it's a `builder`.

**Review by narcodico — CHANGES_REQUESTED** [review]

File: lib/pages/LandingPage.dart (hunk 1 above)
Comment (Friendlygeckos) [reply lib/pages/LandingPage.dart]: Done, I created custom widgets and separated them into different files

File: lib/pages/landingPage/components/start_trading_button.dart [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        style: ButtonStyle(
+          shape: MaterialStateProperty.all<RoundedRectangleBorder>(
+            RoundedRectangleBorder(
+              borderRadius: BorderRadius.circular(100),
+              side: BorderSide(color: Colors.amber[400]!)
+            )
+          )
+        ),
+        onPressed: () {
+          Navigator.push(
```
Comment (narcodico) [inline lib/pages/landingPage/components/start_trading_button.dart:27]: Extract this handler into a reusable function.

File: lib/pages/landingPage/components/android_landing_page.dart [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import 'package:flutter/material.dart';
+
+class AndroidLandingPage extends StatelessWidget {
```
Comment (narcodico) [inline lib/pages/landingPage/components/android_landing_page.dart:3]: This naming should refer to mobile not android.

File: lib/pages/landingPage/LandingPage.dart [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+                //AX Markets Image
+                AthleteXLogo(height: _height),
+                DesktopLandingPage(textSize: textSize),
+                //Button load athletes
+                Container(
+                    width: _width * 0.20,
+                    height: _height * 0.1,
+                    margin: EdgeInsets.only(bottom: 130.0),
+                    child: StartTradingButton(
+                        isWeb: isWeb, tradingTextSize: tradingTextSize)),
```
Comment (narcodico) [inline lib/pages/landingPage/LandingPage.dart:48]: Use commas to properly format widgets.

**Review by narcodico — CHANGES_REQUESTED** [review]

**Review by narcodico — APPROVED** [review]

**Pull request merged**